homeassistant/components/samsungtv/quality_scale.yaml

```diff
@@ -0,0 +1,102 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: no custom actions
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow:
+    status: todo
+    comment: cleanup "pin" vin CONF_PIN
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: no actions
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup:
+    status: exempt
+    comment: no events
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
+
+  # Silver
+  action-exceptions:
+    status: todo
+    comment: handle _async_send_keys and async_set_volume_level exceptions
+  config-entry-unloading: done
+  docs-configuration-parameters:
+    status: exempt
+    comment: no configuration options so far
+  docs-installation-parameters: done
+  entity-unavailable:
+    status: todo
+    comment: check super().unavailable
+  integration-owner: done
+  log-when-unavailable: done
+  parallel-updates: done
+  reauthentication-flow: done
+  test-coverage: done
```

Comment:
It would be nice to have `setup_samsungtv_entry` actually get a real life like unique id

Reply:
The typehint `ConfigType` is not correct IMO as we always use that for YAML config, not for entry data